google-cloud-logging/lib/google/cloud/logging/v2/doc/google/logging/v2/logging.rb

```diff
@@ -19,9 +19,9 @@ module Logging
     #
     # | Class | Description |
     # | ----- | ----------- |
-    # | [LoggingServiceV2Client][] | The Stackdriver Logging API lets you write log entries and manage your logs, log sinks and logs-based metrics. |
-    # | [ConfigServiceV2Client][] | The Stackdriver Logging API lets you write log entries and manage your logs, log sinks and logs-based metrics. |
-    # | [MetricsServiceV2Client][] | The Stackdriver Logging API lets you write log entries and manage your logs, log sinks and logs-based metrics. |
+    # | [LoggingServiceV2Client][] | Writes log entries and manages your Stackdriver Logging configuration. |
+    # | [ConfigServiceV2Client][] | Writes log entries and manages your Stackdriver Logging configuration. |
+    # | [MetricsServiceV2Client][] | Writes log entries and manages your Stackdriver Logging configuration. |
```

Comment:
Shouldn't the description be different for each client? (Ditto appearances of the same table elsewhere.)

Reply:
Good point. This comes from the [API-level description](https://github.com/googleapis/googleapis/blob/70402cea3d4f76878ebe83907c4ccc1c604b39b7/google/logging/logging.yaml#L12), rather than the [service-level comments](https://github.com/googleapis/googleapis/blob/70402cea3d4f76878ebe83907c4ccc1c604b39b7/google/logging/v2/logging_metrics.proto#L34) on the proto. We should probably use the latter and fall back on the former only when there's no service-level comment available.

Filed https://github.com/googleapis/toolkit/issues/1689 to track.